grails-app/services/org/pih/warehouse/product/ProductService.groovy

```diff
@@ -1088,35 +1088,41 @@ class ProductService {
 
             if (terms) {
                 terms.each { term ->
+                    term = term + "%"
```

Comment:
Do you want to change this behavior from looking anywhere in string to 'starts with' version?

Reply:
Yes. See the ticket for more info.